internal/pkg/v2/application/dto/v2dot0/common/metrics/metrics.go

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+// metrics contains v2.0 metrics request and response DTOs.
+package metrics
+
+import dtoBase "github.com/edgexfoundry/edgex-go/internal/pkg/v2/application/dto/v2dot0/common/base"
+
+// Request defines the input for this use case.
+type Request struct {
+	dtoBase.Request `json:",inline"`
+}
+
+// Response defines the output/result for this use case.
```

Comment:
gdoc for the fields would be helpful... I also was going to suggest as an alternative that a reference to the openapi definition for this DTO would be useful, but the question then is to which one, as this response DTO is defined in each service's openapi yaml file. Maybe we should investigate whether some of these common definitions can be extracted into their own yaml files and then reference by the main service yaml files?